### ISSUE TITLE
A11y: [Send a video] Video controls (Play/Mute) not focusable with exte...

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-android#6399]
- **Checkpoint ID:** 11.2.1.1/10.1
- **Component:** Send a video

### Issue Description
Video controls (Play/Mute) not focusable with external keyboard.

### Recommendation
Ensure all video controls are in the keyboard focus order.

--- 
*Report Date: 25.02.2026*
